ChangeLog.md

```diff
@@ -4,6 +4,11 @@ The format is based on
 and this project adheres to
 [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## v2.13.3 -- not yet released
```

Comment:
```suggestion
## vX.Y.Z -- YYYY-MM-DD
```
I usually use this as the version placeholder because we don't know what kind of version bump the next release will require yet.